cypress/integration/unit_tests/color_function_util_test.js

```diff
@@ -107,35 +275,40 @@ describe('Unit tests for color function utility', () => {
     expect(td.children().first().css('background-color')).to.equal('red');
 
     // switch to continuous
-    $('#CONTINUOUS-radio').trigger('change');
+    const propertyRadio = $('#property-radio');
+    propertyRadio.prop('checked', true).trigger('change');
     expectOneFirebaseWrite();
-    const continuousPropertyPicker = $('#continuous-property-picker');
+
+    const continuousRadio = $('#CONTINUOUS-radio');
+    expect(continuousRadio).to.have.prop('checked', true);
```

Comment:
nit: this is fine, but I think you can just say `expect(continuousRadio).not.to.be.checked`. Here and below